src/lib/urn.ts

```diff
@@ -34,7 +34,7 @@ const protocolMatcher = '(?<protocol>mainnet|ropsten|matic|mumbai|off-chain)'
 const typeMatcher = '(?<type>base-avatars|collections-v2|collections-thirdparty)'
 
 const baseAvatarsSuffixMatcher = '((?<=base-avatars:)BaseMale|BaseFemale)'
-const collectionsSuffixMatcher = '((?<=collections-v2:)0x[a-fA-F0-9]{40})'
+const collectionsSuffixMatcher = '((?<=collections-v2:)(?<collectionAddress>0x[a-fA-F0-9]{40}))?(:(?<tokenId>[^:|\\s]+))'
```

Comment:
I think this is missing an `=` after each `<`
```suggestion
const collectionsSuffixMatcher = '((?<=collections-v2:)(?<=collectionAddress>0x[a-fA-F0-9]{40}))?(:(?<=tokenId>[^:|\\s]+))'
```

And that might be breaking the tests

Reply:
The equal sign matches there the exact string (at least as far as I understood), but that's not needed for the collectionAddress nor for the tokenId.

Reply:
Correct! my bad